lib/index.js

```diff
@@ -54,17 +55,17 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
         account, email, hash, clientId, serviceId,
         authenticationMethod, requiredAuthenticationMethods
       } = req.body;
-
       if(type === 'nonce') {
-        // permitted without authentication
+        // permitted without authentication. Do not require an account since
+        // anyone can request for a nonce.
```

Comment:
Our comment style is that if we need full punctuation, capitalization, and complete sentences/paragraphs, that sort of thing -- to describe what's going on, we should do a full multiline comment (`/* Some sentences and a length explanation goes here. */`). If not, we do the simpler inline comments that don't need all that. Since this style mixed the two together, I made the suggestion below. I also tweaked the language a little bit to remove the implication 

```suggestion
        // since anyone can request a nonce, this is permitted without authentication
```